docs/g_slide_012.md

```diff
@@ -7,4 +7,4 @@
 ## *This work was supported by the National Science Foundation under the Emerging Frontiers in Research and Innovation (EFRI) program (EFMA-1830870).*
 
 {:style="text-align:center;"}
-[Previous slide](https://rivesunder.github.io/yuca/g_slide_010) -- [References](https://rivesunder.github.io/yuca/g_ref)
+[Previous slide](https://rivesunder.github.io/yuca_docs/g_slide_010) -- [References](https://rivesunder.github.io/yuca_docs/g_ref)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.